websites/H/Hexa Watch/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.13",
+  "apiVersion": 1,
+  "author": {
+    "name": "Anant",
+    "id": "629851641579044874"
+  },
+  "service": "Hexa Watch",
+  "description": {
+    "en": "Hexa Watch is a streaming platform for movies and TV shows. Watch your favorite content with ease!",
+    "fr": "Hexa Watch est une plateforme de streaming pour les films et les émissions de télévision. Regardez votre contenu préféré en toute simplicité!"
+  },
+  "url": "www.hexa.watch",
+  "regExp": "https?://(www\\.)?hexa\\.watch(/.*)?",
+  "version": "1.0.0",
+  "logo": "https://i.ibb.co/6c2kY3tv/icon.png",
+  "thumbnail": "https://i.ibb.co/fG1XPK2B/Screenshot-2025-05-09-102554.png",
```

Comment:
This screenshot is a picture of the Discord activity, not a promotional image from the website or a screenshot of the website.